src/dash-playlist-loader.js

```diff
@@ -766,10 +764,11 @@ export default class DashPlaylistLoader extends EventTarget {
 
                 // Clear & reset timeout with new minimumUpdatePeriod
```

Comment:
It may be worth created a `updateMinimumUpdatePeriodTimeout_()` function to centralize the logic of the three uses (even can include the clearing) and add a comment around why we use the target duration when the minimumUpdatePeriod is 0.

Reply:
Yup makes sense. Done